### ISSUE TITLE
Improve doc on dependencies

### ISSUE THREAD

**Issue by wembernard** [npm/docs#763]
Hi there,

It would be very useful to add this explanation from StackOverflow on the subject: http://stackoverflow.com/a/22004559/689429

This actual documentation about the subject is not enough 👍

**Comment by ashleygwilliams** [comment]
yes! i actually discovered i have an old PR open for that here https://github.com/npm/docs/pull/732

i'll take a look at that stack overflow answer and finish up that PR soon! thanks!

**Comment by CarolynWriter** [comment]
Reinvestigating this issue .
